spec/models/manageiq/providers/hawkular/middleware_manager/middleware_datasource_spec.rb

```diff
@@ -1,13 +1,15 @@
 require_relative 'hawkular_helper'
 
+# VCR Cassettes: Hawkular Services 0.0.8.Final-SNAPSHOT (commit 0b50b06025641fa83128bb13de3a3eff8d37bb8b)
+
 describe ManageIQ::Providers::Hawkular::MiddlewareManager::MiddlewareDatasource do
 
   let(:ems_hawkular) do
     _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
     auth = AuthToken.new(:name => "test", :auth_key => "valid-token", :userid => "jdoe", :password => "password")
     FactoryGirl.create(:ems_hawkular,
-                       :hostname        => 'localhost',
-                       :port            => 8080,
+                       :hostname        => 'hservices.torii.gva.redhat.com',
```

Comment:
Would you mind putting host/port also into the `hawkualr_helper.rb` as you did with the start/end time, so that we have one stop shopping for this?